src/components/CallToAction/__tests__/CallToAction.unit.test.tsx

```diff
@@ -1,9 +1,9 @@
-import { render } from "../../../testUtils";
+import { render, screen } from "../../../testUtils";
 import CallToAction from "../CallToAction";
 
 describe("<CallToAction/>", () => {
-  it("should render component", () => {
-    const { container } = render(<CallToAction setMigrationModalOpen={() => console.log("setMigrationModalOpen")} />);
-    expect(container).toMatchSnapshot();
+  it("should display CTA", () => {
+    render(<CallToAction setMigrationModalOpen={() => console.log("setMigrationModalOpen")} />);
+    expect(screen.getByText("You have assets ready to migrate to v2"));
```

Comment:
this snapshot feels small enough to keep?

Reply:
im good w/ keeping this. will revert